examples/notebooks/keras-classification/b - Secure Classification with TFE Keras - Secure Model Serving .ipynb

```diff
@@ -16,7 +16,9 @@
      "name": "stderr",
```

Comment:
<p>imports could be cleaned up</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tf-encrypted/tf-encrypted/pull/642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>ah good catch!</p><p> </p>